tests/blotter/test_core.py

```diff
@@ -0,0 +1,119 @@
+from zipline.finance.blotter import SimulatedBlotter
+from zipline.finance.blotter.core import BlotterClassDispatcher
+from zipline.testing.fixtures import ZiplineTestCase
+from zipline.testing.predicates import (
+    assert_equal,
+    assert_is,
+    assert_raises_str,
+    assert_true,
+)
+from zipline.utils.compat import mappingproxy
+
+bcd = BlotterClassDispatcher(blotter_factories={})
+
+
+class BlotterCoreTestCase(ZiplineTestCase):
+
+    def init_instance_fixtures(self):
+        super(BlotterCoreTestCase, self).init_instance_fixtures()
+        bcd.clear()
```

Comment:
instead of clearing this every time, why don't we just create a new one to be safe?